pom.xml

```diff
@@ -235,6 +235,7 @@
                      <exclude>parquet/org/**</exclude>
                      <exclude>parquet/column/**</exclude>
                      <exclude>parquet/hadoop/ParquetInputSplit</exclude>
+                     <exclude>parquet/thrift/struct/**</exclude>
```

Comment:
In this instance the most important thing is to keep compatibility regarding deserializing old JSON that did not have this field. It sounds like the old  constructor is not necessary. I don't feel strongly about this. But the unknown case will happen anyway so maybe keep the old constructor deprecated? Do we need to remove the annotation anyway?